internal/website/content/howto/docstore/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Docstore"
+date: 2019-06-08T15:11:57-04:00
+draft: true
+showInSidenav: true
+---
+
+Docstore provides an abstraction layer over common document stores like Amazon
+DynamoDB, Google Cloud Firestore, and MongoDB. These guides show how to work
```

Comment:
Consider adding a link to a definition of Document Store? Or do you think that the examples are sufficient?

Reply:
Done.